### PR TITLE
fix: Do not show icon and notifications anymore.

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes https://github.com/fossasia/open-event-frontend/issues/7433

#### Short description of what this resolves:
Hides notification pannel for now.

#### Changes proposed in this pull request:
Screenshot from 2021-06-15 10-43-35

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/notification-dropdown.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<i class="mail outline icon {{unless this.unreadNotifications.length 'ui no right margin'}}"></i>
+{{!-- <i class="mail outline icon {{unless this.unreadNotifications.length 'ui no right margin'}}"></i>
```
Comment (maze-runnar) [inline app/templates/components/notification-dropdown.hbs:1]: just change in nav-bar.hbs

Comment (iamdevvalecha) [reply app/templates/components/notification-dropdown.hbs:1]: ok

**Comment by MeghalBisht** [comment]
This just removes the notification icon.
I can still access Notification at `https://open-event-frontend-pkvifxx67-eventyay.vercel.app/notifications/unread`.

File: app/templates/components/notification-dropdown.hbs (hunk 1 above)
Comment (iamdevvalecha) [reply app/templates/components/notification-dropdown.hbs:1]: @SundaramDubey just changing in nav-bar does'nt resolves problem. i have to hide icon and app/notifications/unread
as well.

**Comment by maze-runnar** [comment]
there is still icon space between profile and search icon that was for icon - 
Screenshot (79)

**Comment by maze-runnar** [comment]
test is also failing. please make sure test passes before requesting review.

**Review by maze-runnar — CHANGES_REQUESTED** [review]
complete the changes I have said

**Comment by iamdevvalecha** [comment]
changes:-
Screenshot from 2021-06-17 17-20-29

**Comment by iamdevvalecha** [comment]
@maze-runnar i have done the changes. i am not sure why the test case is failing!

**Review by maze-runnar — APPROVED** [review]

**Review by MeghalBisht — APPROVED** [review]

**Pull request merged**